illumidesk-notebook/requirements.txt

```diff
@@ -1,4 +1,4 @@
-https://github.com/IllumiDesk/nbgrader/archive/v0.64.3.zip
+https://github.com/IllumiDesk/async_nbgrader/archive/v0.1.0.zip
```

Comment:
@rupeshparab update to v0.1.1 based on the [latest release](https://github.com/IllumiDesk/async_nbgrader).

Reply:
updated release version